wolfssl/test.h

```diff
@@ -52,18 +52,19 @@
     #include <string.h>
     #include "rl_net.h"
     #define SOCKET_T int
-        typedef int socklen_t ;
-        static unsigned long inet_addr(const char *cp)
+    typedef int socklen_t ;
+    #define inet_addr wolfSSL_inet_addr
```

Comment:
Why was this define introduced (mapping inet_addr to wolfSSL_inet_addr)?  I don't see that it ever gets defined to anything else.

Reply:
MDK has inet_addr, but has mismatched not cast-able return value type. 
wolfSSL_inet_addr is for avoiding the conflict,  providing functionally 
compatible to standard inet_addr.

Reply:
Ok, thanks